requirements.txt

```diff
@@ -0,0 +1,65 @@
+asgiref==3.5.2
+attrs==21.4.0
+bcrypt==3.2.2
+beautifulsoup4==4.9.3
+black==22.1.0
+certifi==2021.10.8
+cffi==1.15.0
+chardet==4.0.0
+charset-normalizer==2.0.12
+click==8.0.4
+colorama==0.4.4
+cryptography==37.0.2
+distro==1.7.0
+Django==4.0.4
+docker==5.0.3
+docker-compose==1.29.2
+dockerpty==0.4.1
+docopt==0.6.2
+flake8==4.0.1
+idna==3.3
+iniconfig==1.1.1
+joblib==1.1.0
+jsonschema==3.2.0
+mccabe==0.6.1
+mypy-extensions==0.4.3
+nltk==3.7
+packaging==21.3
+paramiko==2.11.0
+pathspec==0.9.0
+platformdirs==2.5.1
+pluggy==1.0.0
+py==1.11.0
+psycopg2-binary==2.9.3
+pycodestyle==2.8.0
+pycparser==2.21
+pyflakes==2.4.0
+PyNaCl==1.5.0
+pyparsing==3.0.9
+pyrsistent==0.18.1
+pytest==7.1.2
+pytest-watch==4.2.0
+python-dotenv==0.20.0
+PyYAML==5.4.1
+regex==2022.4.24
+requests==2.27.1
+six==1.16.0
+soupsieve==2.3.2.post1
+sqlparse==0.4.2
+texttable==1.6.4
+tomli==2.0.1
+tqdm==4.64.0
+typing_extensions==4.2.0
+urllib3==1.26.9
+watchdog==2.1.6
+websocket-client==0.59.0
+
+pip~=22.0.4
+wheel~=0.37.1
+google~=3.0.0
+Pillow~=9.1.1
+pytz~=2022.1
+numpy~=1.22.3
+setuptools~=58.1.0
+djangorestframework~=3.13.1
+Markdown~=3.3.7
```

Comment:
I was experiencing issues getting the image to build with some of these packages. I didn't sift through which were causing the errors though, and apparently building on M1 is a little finicky so it could just be a problem on my end. In any case, if you're still getting errors after fixing the isAuthenticated bug commented on above, try replacing the packages here with the ones from the course repo:

```
asgiref==3.5.0
black==22.1.0
click==8.0.4
Django==4.0.3
djangorestframework==3.13.1
flake8==4.0.1
mccabe==0.6.1
mypy-extensions==0.4.3
pathspec==0.9.0
platformdirs==2.5.1
psycopg2-binary==2.9.3
pycodestyle==2.8.0
pyflakes==2.4.0
pytz==2021.3
sqlparse==0.4.2
tomli==2.0.1
```